README.md

```diff
@@ -86,107 +86,104 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 
 ## 📋 Requirements
 
-- **🍎 Apple M1/M2/M3 Mac or 🐧 Linux system** (tested on Fedora).
-  We anticipate support for more operating systems in the future.
+- **🍎 Apple M1/M2/M3 Mac or 🐧 Linux system** (tested on Fedora). We anticipate support for more operating systems in the future.
+
+   📋  When installing the `ilab` CLI on macOS, you may have to run the `xcode-select --install` command, installing the required packages previously listed.
+
 - C++ compiler
 - Python 3.10 or Python 3.11
-- Approximately 60GB disk space (entire process)
 
-> **NOTE:** Python 3.12 is currently not supported, because some dependencies don't work on Python 3.12, yet.
-<!-- -->
-> **NOTE:** When installing the `ilab` CLI on macOS, you may have to run the `xcode-select --install` command, installing the required packages previously listed.
+   📋 Python 3.12 is currently not supported. Some InstructLab dependencies don't work on Python 3.12, yet.
 
-## ✅ Getting started
+- Approximately 60GB disk space (entire process)
 
-### 🧰 Installing `ilab`
+## ✅ Getting started
 
-1. When installing on Fedora Linux, install C++, Python 3.10 or 3.11, and other necessary tools by running the following command:
+- When installing on Fedora Linux, install C++, Python 3.10 or 3.11, and other necessary tools by running the following command:
 
    ```shell
    sudo dnf install gcc gcc-c++ make git python3.11 python3.11-devel
    ```
 
-   If you are running on macOS, this installation is not necessary and you can begin your process with the following step.
+   Some Python version management tools that build Python (instead of using a pre-built binary) may not by default build libraries implemented in C, and CPython when installing a Python version. This can result in the following error when running the `ilab data generate` command: `ModuleNotFoundError: No module named '_lzma'`. This can be resolved by building CPython during the Python installation with the `--enable-framework`. For example for `pyenv` on MacOS: `PYTHON_CONFIGURE_OPTS="--enable-framework" pyenv install 3.x`. You may need to recreate you virtual environment after reinstalling Python.
 
-2. Create a new directory called `instructlab` to store the files the `ilab` CLI needs when running and `cd` into the directory by running the following command:
+> [!NOTE]
+> The following steps in this document use [Python venv](https://docs.python.org/3/library/venv.html) for virtual environments. However, if you use another tool such as [pyenv](https://github.com/pyenv/pyenv) or [Conda Miniforge](https://github.com/conda-forge/miniforge) for managing Python environments on your machine continue to use that tool instead. Otherwise, you may have issues with packages that are installed but not found in your virtual environment.
+
+### 🧰 Installing `ilab`
+
+1. Create a new directory called `instructlab` to store the files the `ilab` CLI needs when running and `cd` into the directory by running the following command:
```

Comment:
No longer true - we have dedicated directories for `ilab` at the system level now - @cdoern and @RobotSail can share details

Reply:
There's also some detail about this further down in the doc

Reply:
Yeah I didn't have to do this either, so Im good with removing it. I just wan't sure if it was a organization thing.